### PR TITLE
Fixes wing examine text

### PR DESCRIPTION
**Pull request by Mqiib**
"He has also have a pair of angel/devil wings on their back" 

# Changelog

:cl:  

spellcheck: Fixes wing examine text to not be weird  

/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -117,7 +117,7 @@
 		var/badwings = ""
 		if(mind?.martial_art && istype(mind.martial_art, /datum/martial_art/ultra_violence))
 			badwings = "Weaponized "
-		. += "[t_He] [t_has] also have a pair of [span_warning(badwings)][(dna.features["wings"])] wings on their back"
+		. += "[t_He] [t_has] a pair of [span_warning(badwings)][(dna.features["wings"])] wings on [t_his] back"
```
Comment (MajManatee) [inline code/modules/mob/living/carbon/human/examine.dm:120]:
```suggestion
		. += "[t_He] [t_has] also has a pair of [span_warning(badwings)][(dna.features["wings"])] wings on [t_his] back"
```
thoughts?

Comment (Mqiib) [reply code/modules/mob/living/carbon/human/examine.dm:120]: "He has also has a pair of wings"

Comment (MajManatee) [reply code/modules/mob/living/carbon/human/examine.dm:120]: fug

**Review by MajManatee — APPROVED** [review]
I cant spell, do not judge me

**Pull request merged**